vllm_ascend/model_runner.py

```diff
@@ -560,7 +561,7 @@ def _compute_lens(self, inter_data: InterDataForSeqGroup, seq_idx: int,
             context_len = seq_data.get_num_computed_tokens()
             seq_len = min(seq_len, context_len + token_chunk_size)
         elif self.runner.scheduler_config.is_multi_step or \
-            self.runner.model_config.is_encoder_decoder:
+            self.is_encoder_decoder:
```

Comment:
are there any other same changes should also be changed? such as `is_multi_step` above?